rq/utils.py

```diff
@@ -26,6 +27,12 @@
 logger = logging.getLogger(__name__)
 
 
+class DequeueStrategy(str, Enum):
```

Comment:
Can we just use plain `Enum`? No need to subclass from string.

Reply:
Also, can we move `DequeueStrategy` to `constants.py`?

Reply:
> Can we just use plain Enum? No need to subclass from string.

The reason for subclassing is to avoid having to instantiate the `Enum`, and just use a string directly when using as an input - same for output, no need to use `.value` (I think `mypy` also complains about the raw `Enum`). I can make that change anyways.

> Also, can we move DequeueStrategy to constants.py?

Sure, agree it would be a better place. We don't have a `constants` yet though - if we were to add this, it would probably make sense to concentrate other constants as well for consistency. My suggestion would be to make this change in a new PR.

Reply:
Plain Enum: ok. Let me know if we should adopt the `constants` now or on the next iteration.

Reply:
> Let me know if we should adopt the constants now or on the next iteration.

Actually you're right, we don't yet have `constants.py`, what we have is `defaults.py` which is not the right place to put this.

Maybe just move this into `worker.py`?

Reply:
done